flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is a disappointment.

Reply:
The manifest is using the binary release of my application and I don't currently have the resources to compile for arm.

Reply:
why don't you let flathub build it. like it should be.

Reply:
I am working on having the manifest build it from source so it should build on arm.

Reply:
The manifest now builds it from source, allowing for arm support.